exonum-java-binding/core/src/test/java/com/exonum/binding/core/runtime/ServiceArtifactIdTest.java

```diff
@@ -41,25 +41,30 @@ void parseFromRoundtrip(String serviceId) {
   void valueOf() {
     int runtimeId = 1;
     String name = "test-service";
-    ServiceArtifactId id = ServiceArtifactId.valueOf(runtimeId, name);
+    String version = "1.0";
+    ServiceArtifactId id = ServiceArtifactId.valueOf(runtimeId, name, version);
 
     assertThat(id.getRuntimeId()).isEqualTo(runtimeId);
     assertThat(id.getName()).isEqualTo(name);
+    assertThat(id.getVersion()).isEqualTo(version);
   }
 
   @Test
   void newJavaId() {
-    String name = "test-service";
-    ServiceArtifactId id = ServiceArtifactId.newJavaId(name);
+    String serviceName = "test-service";
```

Comment:
That is not a usual Java service name — it must include groupId, separated with some other delimiter.